src/types/common.d.ts

```diff
@@ -0,0 +1,15 @@
+export interface IMusicAccidental {
+  note: number;
+  accidental: number;
+}
+
+export interface IMusicNoteParts {
+  root: string;
+  accidental: string;
+  type?: string;
+}
+
+export interface IMusicNoteValue {
```

Comment:
Expanding on my comment above. How about:

```javascript
type KeyValue = number; // 0 - 11
type RootValue = number; // 0 - 6
type AccidentalValue = number; // -2  to +2

export interface Key {
  root_index: RootValue;
  int_val: KeyValue;
}
```

Reply:
What about using comments  like?

`//** Range 0 - 11 */`
`type KeyValue = number;`

in order to get it on editor when placing the mouse over the definition
![image](https://user-images.githubusercontent.com/22865285/111107944-26d45d00-8558-11eb-972d-ce3516332191.png)

Reply:
I went for the enumerated values suggested by @wassertim rather than the comments